src/cpp/r/RUtil.cpp

```diff
@@ -254,6 +256,15 @@ bool isPackageAttached(const std::string& packageName)
    return false;
 }
 
+unsigned int activeCodePage()
+{
+#ifdef _WIN32
+   return localeCP;
+#else
+   return 0u;
```

Comment:
Nit: It looks like we formerly used `CP_ACP` as the default codepage on non-Windows platforms, and on Windows when we couldn't get the codepage from R. It looks like 0 *is* `CP_ACP`; can we use the actual constant here for clarity?

Reply:
This is in the non-Windows case, so using `0u` here is just to avoid having to re-define `CP_ACP` for non-Windows in that scope. (Previously we did provide that define; this refactor obviates the need for that -- although perhaps we should make that comment explicit)

Reply:
Yes, let's just leave a comment here so it's clear we're using CP_ACP so it's easier to work backwards from the multibyte conversions if we need to.